app/mailers/submit_application_reminder_mailer.rb

```diff
@@ -2,7 +2,11 @@ class SubmitApplicationReminderMailer < GovukNotifyRails::Mailer
   require_relative 'concerns/notify_template_methods'
   include NotifyTemplateMethods
 
-  def notify_provider(application, name, to = support_email_address)
+  def notify_provider(application_id, name, to = support_email_address)
+    # in order to still work for scheduled sidekiq jobs, allow application rather than application_id to be passed in
+    # This can be changed to use just application id once all the scheduled mailer jobs in sidekiq have been delivered.
```

Comment:
Should this comment be deleted?  I can't see Application objects being passed in anymore

Reply:
yes it should, i have removed it now👍